wbooks/src/app/screens/DetailBook/index.tsx

```diff
@@ -1,9 +1,13 @@
-import React, { ReactElement } from 'react';
-import { Image, Text, View } from 'react-native';
+import React, { ReactElement, useRef, useState } from 'react';
+import { Image, Text, View, Animated, Pressable } from 'react-native';
 import notImageFound from '@assets/General/not_image_found.png';
+import { cyan, green } from '@constants/colors';
 
 import styles from './styles';
 
+const ADD_WISH = 'Add to whislist';
+const CHECK = '✅';
+const EMPTY = '';
```

Comment:
i think all of these should be at a constants file.

path: screens/DetailBook/constants.ts